Fix #100: Add pylint in workflow

SD-13 opened the pull request:
Fix #100 
Adds pylint in the workflow to lint all python files in this repo.

SD-13 committed:
add pylint in workflow

SD-13 committed:
nits

U8NWXD commented on .github/workflows/linter.yml:
```diff
@@ -10,11 +10,15 @@ on:
       - name: Lint markdown files
         run: |
           python scripts/linter.py
+      - name: Pylint python files
```
Can we make this a separate job? That way devs can easily see whether the Markdown or Python linter is failing

SD-13 replied:
Yeah, added it as a separate job

U8NWXD commented on scripts/linter.py:
```diff
@@ -19,7 +21,9 @@ def find_regex_in_file_content(
             for i, line in enumerate(file):
                 if search_pattern.search(line):
-                    print(f'{filename}:{i+1}: error: wrong notation "{search_pattern.search(line).string}"; expected "{correct_string}"')
+                    print(f"{filename}:{i+1}: error: wrong notation "
```
Since there are going to be indented lines below, it's easier to read when they're all together as a block

```suggestion
                    print(
                        f"{filename}:{i+1}: error: wrong notation "
```

SD-13 replied:
done

SD-13 committed:
resolve conflict + some changes

SD-13 committed:
nit

SD-13 commented:
Hi @U8NWXD , I have addressed all your comments and also added the pylint in the wiki, Please let me if it is in the right place or needs any other changes. Thanks,

SD-13 commented:
@U8NWXD I know you are busy but just making sure you didn't missed it

U8NWXD commented on Wiki.md:
```diff
@@ -84,6 +85,21 @@ Then for every new contribution (including your first), you should follow these
 9. Once reviewers have approved, the welfare team will merge your PR, and your changes will be automatically deployed to the Oppia wiki. Congratulations!
 
+### Run pylint on python files in the [Oppia-docs-repo](https://github.com/oppia/oppia-web-developer-docs).
+steps:
```
```suggestion
Steps:
```

SD-13 replied:
Updated

U8NWXD commented on Wiki.md:
```diff
@@ -84,6 +85,21 @@ Then for every new contribution (including your first), you should follow these
+  1.  Install Pylint
+
+      If pip refers to python3, then run:
+      > pip install pylint
```
Can you make these code blocks instead of block quotes? ditto below

SD-13 replied:
Done

U8NWXD commented:
Sorry for the delay, and thanks for the reminder!

SD-13 committed:
small changes

SD-13 committed:
Merge remote-tracking branch 'upstream/develop' into pylint_for_docs_repo

SD-13 commented:
Hey @U8NWXD PTAL. Thanks,
